### PR TITLE
Lang edits to sect 1 - virt env episode

### PR DESCRIPTION
**Pull request by sstevens2**
Working on putting together this PR with small language changes in the [Section 1 virtual environments episode](https://carpentries-incubator.github.io/python-intermediate-development/12-virtual-environments/index.html)

### CODE REVIEW

File: _episodes/12-virtual-environments.md [hunk 1]
```diff
@@ -62,9 +61,9 @@ So what exactly are virtual environments, and why use them?
 
 A Python virtual environment is an **isolated working copy** of a specific version of
 Python interpreter together with specific versions of a number of external libraries installed into that
-virtual environment. A virtual environment is simply a *directory with a particular
-structure* which includes links to and enables multiple side-by-side installations of
-different Python interpreters or different versions of the same external library to coexist on your machine and only one to be selected for each of our projects. This allows you to work on a particular
+virtual environment. 
+A virtual environment package manager enables multiple side-by-side installations of different Python interpreters or different versions of the same external library to coexist on your machine. 
+The virtual environment itself is a *directory with a particular structure* which includes links to those python and package versions allows only one to be selected for each of your projects. This allows you to work on a particular
```
Comment (sstevens2) [inline _episodes/12-virtual-environments.md:66]: I found the original sentence a bit packed with info and the structure was hard to parse. I tried to break it into parts to clarify and ended up braking up the virtual environment package manger vs virtual environment itself

**Comment by sstevens2** [comment]
Some small language suggestions.

File: _episodes/12-virtual-environments.md (hunk 1 above)
Comment (anenadic) [reply _episodes/12-virtual-environments.md:66]: How about: "A Python virtual environment helps us create an **isolated working copy** of a software project that uses a specific version of Python interpreter together with specific versions of a number of external libraries installed into that virtual environment. Python virtual environments are implemented as *directories with a particular structure* within software projects, containing links to specified dependencies allowing isolation from other software projects on your machine that may require different versions of Python and/or external libraries."

Comment (sstevens2) [reply _episodes/12-virtual-environments.md:66]: I think your re-wording sounds great!  I'll edit the PR

**Review by anenadic — APPROVED** [review]

**Review by anenadic — APPROVED** [review]
